deepset_cloud_sdk/workflows/sdk.py

```diff
@@ -0,0 +1,87 @@
+"""Deepset Cloud SDK main class."""
+import structlog
+
+from deepset_cloud_sdk.workflows.pipeline_client.pipeline_service import (
+    _enable_publish_to_deepset,
+)
+
+logger = structlog.get_logger(__name__)
+
+
+class DeepsetSDK:
+    """Main class for Deepset Cloud SDK functionality.
+
+    This class provides a centralized way to initialize and manage SDK features.
+
+    Example for publishing an index or pipeline:
+        ```python
+        from deepset_cloud_sdk.workflows import DeepsetSDK
```

Comment:
please update "publish" to "import" everywhere

Reply:
changed to import_into_deepset